### DISCUSSION TITLE
Ganache Discussions

### DISCUSSION THREAD

**Discussion by emilyJLin95** [ConsenSys-archive/truffle#5121] [Announcements]
If you have Ganache specific questions, please see the discussions on the [Ganache repo](https://github.com/trufflesuite/ganache/discussions)

**Comment by iamsuvodeep** [comment]
i want to downlode ganache but it is taking too much time to downlode even though my internet has a good speed

**Comment by asaprosa** [comment]
> Everything is up to date, there is nothing to compile.


getting this on my terminal.....anyone know what's going on?
